perl/README.md

```diff
@@ -0,0 +1,1029 @@
+
+4. **Reserved Delimiters**: The reserved delimiters described in points 1-3 (`;`, `/`, and `|`) will be stripped and replaced with a comma if found in source inputs to the Bystro Annotation Database.
+
+## What Information Can Bystro Annotator Output?
+
+Bystro Annotator is a general-purpose data curation and labeling engine for genetic data, and has no restrictions on the types of annotations/feature labels it can output. Theoretically it can even support binary data, such as images.
+
+## Variant Representations
+
+Bystro's variant representation deviates slightly from the standard VCF format in the name of simplicity. In particular, it drops the rule that the alternate allele must be ACTG. Dropping this single restriction allows us to represent deletions as occuring at the actual first deleted base, rather than the base before, as in the VCF format. This has a number of knock on benefits:
+
+- The `inputRef` (reference base) in Bystro's annotation outputs is always exactly 1 base long
+
+- The `pos` (position) in Bystro's annotation outputs is always the first affected base, except in the case of insertions, where it is the base before the insertion, since the insertion by definition is between two reference bases
+
+- It is possible to represent all multiallelic site using a single reference base, a single position, and a list of alleles
+
+The Bystro Genotype Dosage Matrix, is a columnar dataset, generated for every collection of VCFs submitted. Its first column is the `locus`, which is `chr:pos:ref:alt`. Every column after that is labeled by the sample name, and contains a -1 for missing genotypes, 0 for reference, 1 for a single alternate allele, 2 for 2 alternate alleles (homozygous in a diploid organism), and so on. It can be used for many things, such as to calculate polygenic risk scores.
+
+### Comparing the VCF and Bystro Variant Representations
+
+Below we'll demonstrate how the Bystro Annotator handles different kinds of variants, using some examples. We'll do these demonstartions using the Bystro VCF preprocessor, which is a Go program used by the Bystro Annotator to convert a VCF into a partially annotated tab-separated output. The Bystro VCF preprocessor is installed with Bystro Annotator (see the [INSTALL.md](./INSTALL.md) file for instructions on how to install Bystro Annotator). If you don't have Bystro Annotator installed, you can still run the examples as long as you install the Bystro VCF preprocessor by running `go install github.com/bystrogenomics/bystro-vcf@2.2.3`.
+
+Please note that we are not showing the full Bystro Annotator output described below. We're showing just the first 17 columns of the output, which are the most important for understanding the variant representation and sample genotype handling.
```

Comment:
Maybe here add that it's 17 out of (total number) instead of below.

Reply:
The total number depends on the database used

Reply:
What's the max number we currently have?

Reply:
I'd have to check, it's something like 220